### PR TITLE
Update tutorials.rst

### PR DESCRIPTION
**Pull request by GeorgeStellas**
added brackeys' video in the video tutorials listing

### CODE REVIEW

File: community/tutorials.rst [hunk 1]
```diff
@@ -19,6 +19,7 @@ Some tutorials mentioned below provide more advanced tutorials, e.g. on 3D or sh
 Video tutorials
 ---------------
 
+- `Brackeys <https://youtu.be/LOhfqjmasi0?si=dksdugZQdHhI9C0O>`_ (2D, GDScript)
```
Comment (AThousandShips) [inline community/tutorials.rst:22]: Should be moved down, alphabetical order

Comment (GeorgeStellas) [reply community/tutorials.rst:22]: thanks for the response let me fix that!

**Review by mhilbrunner — COMMENTED** [review]
Thanks! Up to the foundation to decide. IMO, we should wait until there is a bit more content, amazing as this is.

**Comment by skyace65** [comment]
I agree with mhilbrunner. Good tutorial, but we should probably wait for him to make another video before adding him to the list.

**Comment by skyace65** [comment]
Closing this PR as links to individual channels were removed in #10023

**Pull request closed without merging**